modules/alloy/src/main/ts/ephox/alloy/ui/types/ItemTypes.ts

```diff
@@ -57,7 +57,7 @@ export interface NormalItemSpec {
   components?: AlloySpec[];
   dom: RawDomSchema;
   // INVESTIGATE: this might not be right
```

Comment:
Do we still need to investigate this ? :)

Reply:
No idea to be honest and isn't really something that's in scope of this task 😉 It's also why a TODO generally should never exist without a Jira for more context.

Reply:
I imagine I wrote that when looking through the types that had been added, and it didn't look quite right. So if the type is right, it can be removed.

Reply:
Removed these TODO statements as discussed on slack